File: ios/dAIly/livekit/DebugFlags.swift
import Foundation

/// Compile-time debug feature flags.
/// All flags are `false` (and `let`) in Release builds.
/// In DEBUG builds they are `var` and can be toggled at runtime in a debugger.
public enum DebugFlags {
    #if DEBUG
    /// Enable push-to-talk mode (D-07). When true, VoiceSession.setMicrophone(enabled:)
    /// is active. NEVER surfaced in production UI — debug/testing only.
    public static var pttEnabled: Bool = false
    #else
    public static let pttEnabled: Bool = false
    #endif
}

File: android/app/src/main/kotlin/com/daily/android/livekit/DebugFlags.kt
package com.daily.android.livekit

import com.daily.android.BuildConfig

/**
 * Debug feature flags.
 *
 * All flags short-circuit to false in Release builds (T-20-23).
 * D-07 mirror: PTT (push-to-talk) is never surfaced in production UI.
 */
object DebugFlags {
    /// Push-to-talk: surfaced only in DEBUG builds (D-07 mirror — never in production UI).
    val pttEnabled: Boolean = BuildConfig.DEBUG && false  // flip to true locally to enable
}
